src/backend/gulpfile.js

```diff
@@ -1,106 +1,125 @@
-var gulp = require('gulp');
-var uglify = require('gulp-uglify');
-var babel = require('gulp-babel');
-var concat = require('gulp-concat');
-var minify = require('gulp-minify-css');
-var htmlmin = require('gulp-htmlmin');
-var iife = require('gulp-iife');
-var clean = require('gulp-clean');
+/* eslint-disable no-empty-label */
+'use strict';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['use strict' is unnecessary inside of modules. (strict)](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)